src/plugins/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -90,17 +72,17 @@ export const DashboardLinkComponent = ({
   }, [link, dashboardTitle]);
 
   const { tooltipTitle, tooltipMessage } = useMemo(() => {
-    if (error) {
+    if (link.error) {
       return {
         tooltipTitle: DashboardLinkStrings.getDashboardErrorLabel(),
-        tooltipMessage: error.message,
+        tooltipMessage: link.error.message,
       };
     }
     return {
       tooltipTitle: Boolean(dashboardDescription) ? linkLabel : undefined,
-      tooltipMessage: dashboardDescription || linkLabel,
+      tooltipMessage: dashboardDescription ?? linkLabel,
```

Comment:
This should be `||` still, no? That way, if the dashboard description is the empty string, we still show the `linkLabel`